pyproject.toml

```diff
@@ -1,57 +1,81 @@
 [build-system]
-requires = ["flit_core >=2,<3"]
-build-backend = "flit_core.buildapi"
-
-[tool.black]
-target-version = ["py38"]
-line-length = 119
-
-[tool.flit.metadata]
-module = "rnanorm"
-author = "Genialis, Inc."
-author-email = "miha@genialis.com"
-home-page="https://github.com/genialis/rnaseq-normalization/"
-description-file = "README.rst"
-requires-python=">=3.6, <3.11"
 requires = [
-    "numpy",
-    "pandas",
-    "scipy",
-    "pybedtools",
+    "setuptools >= 64.0.0",
```

Comment:
Do you have any special reason why pick this version?
Same goes for the `setuptools_scm` below.

Reply:
[Support for editable installs](https://github.com/pypa/setuptools/blob/main/CHANGES.rst#v6400) with `pyproject.toml` was added at this point.

Reply:
As for `setuptools_scm`, Python3.10 support was added in that version.